packages/apps/plugins/plugin-space/src/SpacePlugin.tsx

```diff
@@ -148,10 +145,14 @@ export const SpacePlugin = (): PluginDefinition<SpacePluginProvides> => {
             }
           case 'presence':
             return SpacePresence;
+          case 'status':
+            return SpaceStatus;
           default:
             return null;
         }
       },
+      // TODO(burdon): Where is "Main" defined?
+      // TODO(burdon): 'components' vs 'component' in graph API?
```

Comment:
- this isn't the graph api, it's the surface api
- `components` is a dictionary of extra components which a plugin can provide and can be referenced by name
- `component` is a function which when passed `data` & `role` returns a component if it can render the `data` in the `role`
- `component` is used more often in our applications
- we use `components` primarily in setting up the application layout so far
- I have a todo somewhere to consider exposing all components that a plugin provides for surfaces via `components`

Reply:
thanks this -- and the docs PR you sent me -- really helps.